Mathematica.java

```diff
@@ -2,4 +2,33 @@
     public static int sum(int a, int b){
         return a + b;
     }
+    public static int power (int a, int b) {
+        int result = a;
+        for (int i = 1; i <= b; i++){
+            result *= a;
+        }
+        return result;
+    }
+    public static int subtract(int a, int b) {
+        return a - b;
+    }
+    public static int multiplication(int a, int b){
+        return a * b;
+    }
```

Comment:
Change method name `division` to `divide` so that it actually sounds like a verb. e.g `Mathematica.divide(4,2)` looks better than `Mathematica.division(8/4)`